Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -784,9 +803,9 @@ function logInfo {
 # In the past we copied to the users temp folder, difficult to find on machines with lots of profiles. Solved this by always placing the log in the normal Huntress folder.
 function copyLogAndExit {
     Start-Sleep 1
-    $logLocation = $DebugLog
-    $agentPath   = getAgentPath
-    Copy-Item -Path $logLocation -Destination $agentPath -Force
+    $agentPath = getAgentPath
+    $logLocation = Join-path $agentPath "HuntressPoShInstaller.log"
```

Comment:
capital P in Join-Path?

Reply:
Should work either way, but just a suggestion

Reply:
Never hurts to make it look better :)